src/main/java/com/microsoft/jenkins/servicefabric/command/SFCommandBuilder.java

```diff
@@ -0,0 +1,176 @@
+/*
+
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+
+ * Licensed under the MIT License. See LICENSE in the project root for
+
+ * license information.
+
+ */
+package com.microsoft.jenkins.servicefabric.command;
+
+import hudson.FilePath;
```

Comment:
Correct me if i am wrong. Why is this showing as a new file ? We can rename the older file so that we can retain history right ?

Reply:
There isn't a `move` or `rename` operation in Git. It infers the action automatically. We cannot enforce that.